google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionImpl.java

```diff
@@ -208,6 +209,7 @@ static UnitOfWorkType of(TransactionMode transactionMode) {
   private AutocommitDmlMode autocommitDmlMode = AutocommitDmlMode.TRANSACTIONAL;
   private TimestampBound readOnlyStaleness = TimestampBound.strong();
   private QueryOptions queryOptions = QueryOptions.getDefaultInstance();
+  private RpcPriority rpcPriority = RpcPriority.HIGH;
```

Comment:
What I meant regarding this was that it might be good to consider using `null` as the default value (or even change the type of the internal value to the proto enum which also includes the `UNSPECIFIED` value), as the current implementation means that:
1. The client determines what the default value is. However, if no value is sent in the request, the backend can determine what the default value should be.
2. The client always sends a priority value, which makes it impossible to distinguish between clients deliberately setting HIGH, and clients that did not set a value at all.